bears/coffee_script/CoffeeLintBear.py

```diff
@@ -239,6 +241,10 @@ class Foo
         :param allow_empty_functions:
             Allows declaring empty functions.
         """
+        if not max_line_length:
+            MAX_SAFE_INTEGER = 2**53 - 1
+            max_line_length = MAX_SAFE_INTEGER
+
```

Comment:
Please use `sys.maxsize`. Again, see https://github.com/coala/coala-bears/pull/2245/commits for more info.